conf/config.yaml

```diff
@@ -129,6 +129,8 @@ etcd:
     - "http://127.0.0.1:2379"     # multiple etcd address
   prefix: "/apisix"               # apisix configurations prefix
   timeout: 30                     # 30 seconds
+  version: "v3"                   # etcd version: v2/v3
+  api_prefix: "/v3alpha"          # 3.2 -> "/v3alpha", 3.3 -> "/v3beta", 3.4+ -> "/v3"
```

Comment:
we should only support `v3` and `/v3`

Reply:
So we don't need to add configures for that.

Reply:
api_prefix has implemented to be detectd and set by lua code, as changed in [t/APISIX.pm](https://github.com/apache/apisix/blob/97809d8fabf29e78d5470835f8f53888bcb2f23e/t/APISIX.pm#L83). 
I could remove it and move those lua code to etcd_v3.lua if needed.